src/Components/ClusterRules/ClusterRules.cy.js

```diff
@@ -122,6 +122,9 @@ describe('test data', () => {
     expect(filteredData).to.have.length.gte(1);
     expect(filteredData).to.have.length.lt(RULES_ENABLED);
   });
+  it('has 1 rule without impacted field', () => {
+    expect(_.uniq(_.map(data, 'Not available'))).to.have.length.lengthOf(1);
```

Comment:
I do not see any reference to `impacted` here. What is this code suppose to be doing exactly?